docs/integrators/deep-history-squad.md

```diff
@@ -114,6 +117,10 @@ Apart from the flag mentioned above, the setup of a deep-history observer is ide
 Never attach a non-pruned database to a regular observer (i.e. that does not have the above **operation-mode**) - unless you are not interested into the deep-history features. The regular observer irremediably removes, trucates and prunes the data (as configured, for storage efficiency).
 :::
 
+Now that we have finished with the installation part, we can proceed to populate the archive with a non-pruned database. There are two options here:
```

Comment:
we can proceed with populating the database from a non-pruned database archive

Reply:
Done.